### DISCUSSION TITLE
타이머 구현하기

### DISCUSSION THREAD

**Discussion by PH62mem** [naverz/zepeto-studio-kor#798] [5. Q&A]
10..9..8..이렇게 떨어지는 타이머 만드려고합니다.
참고하면서 공부하려는데 관련하여 예제나 참고할만한 코드가 있을까요?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 코루틴이나 델타타임을 활용하시면 됩니다.
10~0까지 카운트다운하는 간단한 예제입니다.

import { Time } from 'UnityEngine';
import { ZepetoScriptBehaviour } from 'ZEPETO.Script'

export default class Countdown extends ZepetoScriptBehaviour {
    
    private count : number = 10;
    private delta : number = 0;
    
    
    Update() {
        if(this.count > -1)
        {
            this.delta += Time.deltaTime;
            if(this.delta >= 1)
            {
                console.log(this.count);
                this.delta = 0;
                this.count--;
            }
        }
    }
}

**Reply by PH62mem** [reply]
정말감사합니다!
